Amazon.IonDotnet/Utils/BitConverterEx.cs

```diff
@@ -0,0 +1,46 @@
+﻿/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+namespace Amazon.IonDotnet.Utils
+{
+    using System.Runtime.CompilerServices;
+
+    internal static class BitConverterEx
```

Comment:
What uses these new utilities (BitConverter, DictionaryExtensions, EncodingExtensions, TaskEx)?

Reply:
Various files within the ion library itself uses these classes.

The reason they didn't show up in the changelog is because all of these new classes used to live in the Compatibility.cs file, which is both no longer needed and bad practice since the file had multiple classes and the classes did not match the file name.

Reply:
Got it, thanks.